.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: php
 install:
-  - pear install -a package.xml
+  - composer install
 php:
-  - 5.5
-script: phpunit test/
\ No newline at end of file
+  - 7.4
+script: phpunit
```

Comment:
If you want to integrate older versions of PHP, [consider this example](https://github.com/pear/PEAR_Exception/blob/1ecf1c7949f05aabd831fc33a6623be658f28250/.travis.yml).

Reply:
Done